packages/orchestration/src/proposals/start-stakeOsmo.js

```diff
@@ -0,0 +1,116 @@
+import { makeTracer } from '@agoric/internal';
+import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+import { E } from '@endo/far';
+import { makeChainHub } from '../utils/chainHub.js';
+
+/**
+ * @import {IBCConnectionID} from '@agoric/vats';
+ * @import {StakeIcaSF} from '../examples/stakeIca.contract';
+ */
+
+const trace = makeTracer('StartStakeOsmo', true);
+
+/**
+ * @param {BootstrapPowers & {
+ *   installation: {
+ *     consume: {
+ *       stakeIca: Installation<
+ *         import('../examples/stakeIca.contract.js').start
+ *       >;
+ *     };
+ *   };
+ * }} powers
+ */
+export const startStakeOsmo = async ({
+  consume: {
+    agoricNames,
+    board,
+    chainStorage,
+    chainTimerService,
+    orchestration,
+    startUpgradable,
+  },
+  installation: {
+    consume: { stakeIca },
+  },
+  instance: {
+    // @ts-expect-error stakeOsmo not typed
+    produce: { stakeOsmo: produceInstance },
+  },
+}) => {
+  const VSTORAGE_PATH = 'stakeOsmo';
+  trace('startStakeOsmo');
+  await null;
+
+  const storageNode = await makeStorageNodeChild(chainStorage, VSTORAGE_PATH);
+  const marshaller = await E(board).getPublishingMarshaller();
+
+  // TODO add osmo to bank
+  const atomIssuer = await E(agoricNames).lookup('issuer', 'ATOM');
+  trace('ATOM Issuer', atomIssuer);
+
+  const chainHub = makeChainHub(await agoricNames);
+
+  const agoric = await chainHub.getChainInfo('agoric');
+  const osmosis = await chainHub.getChainInfo('osmosis');
+  const connectionInfo = await chainHub.getConnectionInfo(
+    agoric.chainId,
+    osmosis.chainId,
+  );
```

Comment:
this might be nice as,
```suggestion
  const [agoric, osmosis, connectionInfo] = await chainHub.getChains('agoric', 'osmosis');
```

should I add that to https://github.com/Agoric/agoric-sdk/pull/9494 ?

Reply:
Yes, that'd be a nice addition. I think a separate PR works as well